_docs_integrate/delete-identities.md

```diff
@@ -0,0 +1,111 @@
+As the Backbone Operator must ensure that the creator of the support ticket or the writer of the e-mail is actually the Identity whose deletion was requested, the Identity must additionally [approve the IdentityDeletionProcess]({% link _docs_use-cases/use-case-transport-approve-identitydeletionprocess.md %}) afterwards.
+However, the Identity can also [reject the IdentityDeletionProcess]({% link _docs_use-cases/use-case-transport-reject-identitydeletionprocess.md %}) if it has changed its mind about its deletion after the process of Identity deletion was triggered by the Backbone Admin UI.
+Approving or rejecting an IdentityDeletionProcess with `"WaitingForApproval"` as `status` is only possible until the date specified within the `approvalPeriodEndsAt` property has not been exceeded.
+If it is neither approved nor rejected by then, it automatically changes its `status` to `"Cancelled"` and the Identity will not be deleted.
+Moreover, please note that the use cases for approving or rejecting an IdentityDeletionProcess cannot be executed by the Identity if there is no IdentityDeletionProcess with `"WaitingForApproval"` as `status`.
+An error with [code]({% link _docs_integrate/error-codes.md %}) `error.runtime.identityDeletionProcess.noWaitingForApprovalIdentityDeletionProcess` is otherwise thrown.
+After the IdentityDeletionProcess has changed its `status` from `"WaitingForApproval"` to `"Approved"` as a result of the approval, it embodies that the Identity will be deleted after the grace period ends.
+As with the [self-initiated Identity deletion]({% link _docs_integrate/delete-identities.md %}#self-initiated-identity-deletion), the [Cancel IdentityDeletionProcess]({% link _docs_use-cases/use-case-transport-cancel-identitydeletionprocess.md %}) use case can be executed by the Identity within the grace period if it no longer wants to be deleted.
+If it is not cancelled by then, the Identity will be irreversibly deleted from the Backbone.
+
+## Side Effects of Identity Deletion on Relationships
+
+The deletion and even the triggering of the deletion of an Identity logically has an impact on the peers who have [established a Relationship]({% link _docs_integrate/establish-relationships.md %}) with it.
+All peers of the Identity that is currently in deletion are informed that the deletion of the Identity has been triggered.
+This is done via the `transport.peerToBeDeleted` [Connector event]({% link _docs_integrate/connector-events.md %}).
+In addition, it is stored within the `peerDeletionInfo` property of the [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) that the Identity currently has `"ToBeDeleted"` as `deletionStatus`.
+If the Identity is finally deleted, the `deletionStatus` changes to `"Deleted"` and the `transport.peerDeleted` Connector event can be received.
```

Comment:
Can anyone confirm that in this case it is already implemented that the Relationship also changes its `status` to `"DeletionProposed"`? @jkoenig134 @Magnus-Kuhn @Milena-Czierlinski

Reply:
The Backbone has implemented this with release 6.19.0, but afaik we haven't added a test in the Runtime to check the status of the peerDeletionInfo, yet.

Reply:
Thanks for this hint with release 6.19.0 of the Backbone. 🎄

This should already be testable if we can now set the grace period for IdentityDeletionProcesses to zero for our test purposes, right?

Reply:
Yes, however only decreasing the the grace period isn't enough. We also need to actually delete the Identity in the Backbone. We have a [draft PR](https://github.com/nmshd/runtime/pull/329) for that.

Reply:
Alright, I will wait with adding the test for a Relationship being set to `"DeletionProposed"` after the peer is deleted until https://github.com/nmshd/runtime/pull/329 is merged. ☃️

Reply:
I am not sure how actively https://github.com/nmshd/runtime/pull/329 is being worked on. @jkoenig134 ?

Reply:
Ehm, should be ready.